packages/bruno-app/src/components/CollectionSettings/index.js

```diff
@@ -147,9 +147,7 @@ const CollectionSettings = ({ collection }) => {
           Info
         </div>
       </div>
-      <section className={`flex ${['auth', 'script', 'docs', 'clientCert'].includes(tab) ? '' : 'mt-4'}`}>
-        {getTabPanel(tab)}
-      </section>
+      <section className="mt-4 h-full">{getTabPanel(tab)}</section>
```

Comment:
this was the key to the consistent lines placement. otherwise some required extra `<br/>`s etc